### DISCUSSION TITLE
why is it like this now

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#831] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is the readme supposed to look acoustic with its links or it it bugged what happened to the drop-down menus its super duper sad that we got rid of it

**Comment by AshtonDavies** [comment]
It's redesigned, and a lot of people seem to dislike it.

**Comment by 3kh0** [comment]
The dropdowns are not supposed to be used as "sections" per say. Plus this improvement helps with readability.

**Reply by rwg100** [reply, marked as answer]
no it doesn't its extremely racist
